### ISSUE TITLE
Slow dynode (additional data) sync

### ISSUE THREAD

**Issue by beralt85** [duality-solutions/Dynamic#203]
#### Problem: dynode controller wallet sync dynode data
And also, it seems sometimes to get stuck at 100%. It seems to be the more complex the more dynodes you control. This is a problem, because they you might not be able to activate your dynodes fast.

#### Expected behaviour:
dynode data sync should not take more than 5 minutes, if one just opened and closed the controller wallet.

**Comment by daithi-coombes** [comment]
I'm having the same issue. `Synchronizing additional data` takes about 3 - 4 hrs and then appears to hang on 100%. 

```
Dynamic version v1.2.0.0-619e66c-dirty (64-bit)
Linux daithi-desktop 4.4.0-62-lowlatency #83-Ubuntu SMP PREEMPT x86_64 GNU/Linux
```

**Comment by SCDeveloper** [comment]
This is due to the rate of blocks. Will be fixed in the upcoming v1.3.0.0 release.
